src/org/sosy_lab/java_smt/solvers/bitwuzla/BitwuzlaFloatingPointManager.java

```diff
@@ -198,22 +201,36 @@ protected Term fromIeeeBitvectorImpl(Term pNumber, FloatingPointType pTargetType
         pTargetType.getMantissaSize() + 1);
   }
 
+  private String newVariable() {
+    return "__CAST_FROM_BV_" + counter++;
```

Comment:
We should name this utility variable `__javasmt_cast_from_bv_<ctr>` to have some internal naming.

Reply:
done